doc/colvars-refman-main.tex

```diff
@@ -2840,15 +2844,35 @@
 \mathrm{N}^{(n+4)}$ hydrogen bond is defined through a \texttt{hBond}
 colvar component on the same atoms.
 
+
 \begin{cvcoptions}
+  \item %
+  \labelkey{colvar|alpha|prefix}
+  \keydef
+    {prefix}{%
+      \texttt{alpha}}{%
+      Prefix of atom groups to be used for computing the $\alpha$-helix content}{%
+      string}{%
+      \texttt{alpha\_}}{%
+    Atoms involved in an alpha-helix component are based on groups defined within a
+    GROMACS-style index file specified with \refkey{indexFile}{Colvars-global|indexFile} in the global
+    Colvars configuration, i.e. outside of a \texttt{colvar} block.
+    The group names must use a common prefix followed by atom names, e.g. \texttt{alpha\_N}, \texttt{alpha\_CA},
+    and \texttt{alpha\_O} when using the default prefix \texttt{alpha\_}.
+    The CA group is only required when the angle term is enabled, that is when \texttt{hBondCoeff} is less than 1.
+    This option lets users set a different prefix, for example, when several $\alpha$-helical segments
+    must be tracked.}
 
+\cvnamebasedonly{
 \item %
   \labelkey{colvar|alpha|residueRange}
   \key
     {residueRange}{%
       \texttt{alpha}}{%
       Potential $\alpha$-helical residues}{%
     ``$<$Initial residue number$>$-$<$Final residue number$>$''}{%
+    Alternately, in some back-ends like NAMD and VMD, the atoms may be defined using their PSF segment name (segname)
+    and a range of residue numbers.
```

Comment:
Seems a little odd to continue the previous text here

Reply:
Agreed, changed.